runtime/interpreter/value.go

```diff
@@ -2745,12 +2745,7 @@ func getNumberValueMember(interpreter *Interpreter, v NumberValue, name string,
 					other,
 				)
 			},
-			&sema.FunctionType{
```

Comment:
Function types were wrong, the parameters were missing. Reuse the functions type that were already created in the checker